### ISSUE TITLE
submit_type should not be set for subscriptions

### ISSUE THREAD

**Issue by seeARMS** [jofftiquez/vue-stripe-checkout#71]
Using the v3 beta with subscriptions gives the following error in console:
```
POST https://api.stripe.com/v1/payment_pages 400 (Bad Request)

v3:1 Uncaught (in promise) IntegrationError: You cannot use `submit_type` with subscriptions.
    at new t (https://js.stripe.com/v3:1:10518)
    at Ar (https://js.stripe.com/v3:1:42215)
    at e.Ir._handleMessage (https://js.stripe.com/v3:1:47845)
    at e._handleMessage (https://js.stripe.com/v3:1:24449)
    at https://js.stripe.com/v3:1:46447
```

this.submitType should not be set for subscriptions: https://stripe.com/docs/api/checkout/sessions/object#checkout_session_object-submit_type

**Comment by jofftiquez** [comment]
Ack

**Comment by jofftiquez** [comment]
The `submitType` prop is optional. Would mind sharing your code?

**Comment by jofftiquez** [comment]
Hi may I follow up on this? Can you show me your code? Thanks.

**Comment by seeARMS** [comment]
Hi,

Sorry, but I don't have the problematic code anymore. I fixed this by changing the default value of submitType to `null`:

https://github.com/jofftiquez/vue-stripe-checkout/blob/61ca07de61cf2fac08d86d57c971d3ff7ca51a0b/src/Checkout.vue#L28

Even if the prop is optional, it's still being passed with a default value of 'auto' to the `redirectToCheckout` method: https://github.com/jofftiquez/vue-stripe-checkout/blob/61ca07de61cf2fac08d86d57c971d3ff7ca51a0b/src/Checkout.vue#L63

In the Stripe docs for this method, it mentions submitType should not be specified for recurring subscriptions: https://stripe.com/docs/stripe-js/reference#stripe-redirect-to-checkout

By default in your code, it's always specified, with the type 'auto'.

**Comment by jofftiquez** [comment]
Oh I see so that's the problem. Thanks, I will remove the default value then. Thanks.

**Comment by insign** [comment]
Have you removed this? What is the right beta, btw?

**Comment by jofftiquez** [comment]
Hello this has been fixed in `v3.1.3`. Thanks
